### PR TITLE
Makefile: add default defconfig for new board

### PR DESCRIPTION
**Pull request by liuwei987**
- Makefile: add default defconfig for new board

    Tracked-On: #3779
    Signed-off-by: Wei Liu <weix.w.liu@intel.com>

### CODE REVIEW

File: hypervisor/Makefile [hunk 1]
```diff
@@ -48,6 +48,12 @@ override BOARD := $(shell echo `sed -n '/<acrn-config/p' $(BOARD_FILE) | sed -r
 SCENARIO_IN_XML := $(shell sed -n '/<acrn-config/p' $(SCENARIO_FILE) | sed -r 's/.*scenario="(.*)".*/\1/g')
 endif
 
+DEFCONFIG_FILE := $(CURDIR)/arch/x86/configs/$(BOARD).config
+ifneq ($(DEFCONFIG_FILE),)
+$(shell echo CONFIG_BOARD="$(BOARD)" > $(DEFCONFIG_FILE))
+endif
+
```
[the remark this answers is not in the document]
Comment (liuwei987) [reply hypervisor/Makefile]: The file does not exists.

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Comment by jsun26intel** [comment]
suggest add a condition that works only when CONFIG_XML_ENABLED is true,
otherwise the new board.config will be generated in the case user input BOARD=xxx with typo.

**Comment by wenlingz** [comment]
OK to verify

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Pull request merged**